pkgs/development/python-modules/python-nest/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, fetchFromGitHub, lib, python, python-dateutil, requests
+, six, sseclient-py }:
+
+buildPythonPackage rec {
+  pname = "python-nest";
+  version = "4.1.0";
+
+  # pypi does not contain tests, using github sources instead
+  src = fetchFromGitHub {
+    owner = "jkoelker";
+    repo = "python-nest";
+    rev = "b45100440577ca35061e08dda48fddcc49f214eb";
+    sha256 = "0w5iv08pvlb79q078lscd2wpy5i3m3hynl3lkw8pv5l2ik3ln3i1";
+  };
+
+  propagatedBuildInputs = [ python-dateutil requests six sseclient-py ];
+
+  # no tests
+  doCheck = false;
```

Comment:
Tests are important to know early when something breaks. If there are no tests there should be at least a `pythonImportsCheck`.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck